hack/tools/internal/tilt-prepare/main.go

```diff
@@ -957,3 +958,19 @@ func getProviderObj(version *string) func(prefix string, objs []unstructured.Uns
 		return providerObj, nil
 	}
 }
+
+func updateNamespacePodSecurityPolicy(objs []unstructured.Unstructured) {
```

Comment:
I'm going to nit about the function name: It is not [Pod Security Policy](https://kubernetes.io/docs/concepts/security/pod-security-policy/), but [Pod Security Standard](https://kubernetes.io/docs/concepts/security/pod-security-admission/). PSP were removed in k8s 1.25, replaced by the PSS/PSA.